ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2121,7 +2124,7 @@ private enum UnityLayers {
                                       node => node.GetUniqueValue("name"));
       BodyProcessor insert_body = body => {
         Log.Info("Inserting " + body.name + "...");
-        if (!name_to_gravity_model.TryGetValue(
+        if (!name_to_gravity_model!.TryGetValue(
```

Comment:
Change lines 2111 .. 2113 from
```C#
if (name_to_gravity_model == null) {
  Log.Fatal("Cartesian config without gravity models");
}
```
to
```C#
if (name_to_gravity_model == null) {
  throw Log.Fatal("Cartesian config without gravity models");
}
```
so that control flow analysis understands that `name_to_gravity_model` is not null here.